server/services/odata.service.js

```diff
@@ -28,12 +28,33 @@ const PREFER = 'Prefer'
 const ODATA_VERSION_NUMBER = '4.0'
 const PREFER_REPRESENTATION = 'return=representation'
 
+const _replaceUnsafeCharacters = certificateNumber => {
```

Comment:
This method should be at the end of the file along with the other private helper methods (names beginning with _)